components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -93,8 +93,48 @@ class TabsUseCases(
         }
     }
 
+    class RemoveAllTabsUseCase internal constructor(
+            private val sessionManager: SessionManager
+    ) {
+
+        fun invoke() {
+            sessionManager.removeAll()
+
+        }
+    }
+
+    class RemovePrivateTabsUseCase internal constructor(
+            private val sessionManager: SessionManager
+    ) {
+
+        fun invoke() {
```

Comment:
You could consider pass a boolean argument to the invoke function to differ between removing regular/private tabs.

Reply:
ok.will add

Reply:
sure